internal/transcript/azure_blob_sync.go

```
@@ -347,10 +351,61 @@ func (s *AzureBlobSyncer) createAppendBlob(ctx context.Context, name string) err
+	sourceURL := s.endpoint + "/" + s.container + "/" + azureBlobEscapePath(source)
+	req, err := s.newRequest(ctx, http.MethodPut, destination, "", map[string]string{
+		"x-ms-copy-source": sourceURL,
+	}, nil, 0)
+	if err != nil {
+		return err
+	}
+	resp, err := s.do(ctx, req, func() (io.Reader, error) { return nil, nil }, 0)
+	if err != nil {
+		return err
+	}
+	defer resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)